src/ert/_c_wrappers/enkf/enkf_main.py

```diff
@@ -359,19 +362,17 @@ def sample_prior(
         if parameters is None:
             parameters = self._parameter_keys
 
-        for parameter in parameters:
-            config_node = self.ensembleConfig().getNode(parameter)
+        keylist_gen_kw = self.ensembleConfig().get_keylist_gen_kw()
+
+        for parameter in keylist_gen_kw:
             if self.ensembleConfig().getUseForwardInit(parameter):
                 continue
```

Comment:
This continue is no longer hit by the tests. Have behavior changed or is just some test removed?

Reply:
Is now. Unsure why. :+1: